netaddr.go

```diff
@@ -86,6 +86,38 @@ func ParseIP(s string) (IP, error) {
 	return IP{v6}, nil
 }
 
+// IPAddr returns the net.IPAddr representation of an IP. If the IP contains a
+// zone identifier, the Zone field of the returned IPAddr is populated.
+func (ip IP) IPAddr() *net.IPAddr {
+	if ip.ipImpl == nil {
```

Comment:
just put this in the switch below as the first case (case nil ... https://play.golang.org/p/5fMIvT2zg35)